src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+/* #region card */
+.card {
+
```

Comment:
Empty line

Reply:
Look on checklist #3

Reply:
which checklist? can not find it  here https://github.com/mate-academy/layout_product-cards/pull/3 (((